application/tests/CodeSniffer/Standards/Ontowiki/Sniffs/Commenting/FileCommentSniff.php

```diff
@@ -77,14 +78,18 @@ public function process(PHP_CodeSniffer_File $phpcsFile, $stackPtr)
             $git_year1 .='-';
             $git_year1 .=$git_year2;
           }
-          //$year = " * @copyright Copyright (c) " . date('Y') . ", {@link http://aksw.org AKSW}\n";
           $year = " * @copyright Copyright (c) " . $git_year1 . ", {@link http://aksw.org AKSW}\n";
           $this->copyright[4]= $year;
-        }
-        else {
+        } else {
+          echo count($tokens);
```

Comment:
Should this `echo` really be here? Pleas review your changes for left over lines before committing or at least before sending the pull-request.